10308.xml

```diff
@@ -0,0 +1,122 @@
+  <Object ObjectType="MODefinition">
+    <Name>AT&amp;T Connectivity Extension</Name>
+    <Description1><![CDATA[The AT&T Connectivity Extension Object will be used to gather information describing the UE and the operating state of the UE. The AT&T Connectivity Extension Object is an open and standard object used by AT&T.]]></Description1>
+    <ObjectID>10308</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10308</ObjectURN>
+    <LWM2MVersion>1.1</LWM2MVersion>
+    <ObjectVersion />
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+      <Item ID="1">
+        <Name>ICCID</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[The ICCID resource is used to identify the UICC used by the IoT device or module for authentication by the network.]]></Description>
+      </Item>
+      <Item ID="2">
+        <Name>IMSI</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[The IMSI resource is used to log the IMSI used by the UE to identify itself to the network. The IMSI is defined in 3GPP TS 23.003 and is a combination of MCC (Mobile Country Code), MNC (Mobile Network Code), and MSIN (Mobile Subscriber Identification Number).]]></Description>
+      </Item>
+      <Item ID="3">
+        <Name>MSISDN</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[The MSISDN resource will indicate the MSISDN as assigned, if any, to the SIM card. In the case that no MSISDN is assigned to the SIM card the response should be NULL.]]></Description>
+      </Item>
+      <Item ID="4">
+        <Name>APN Retries</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[The APN Retry multi instance resource will maintain the number of retries the device can attempt to activate a context on a particular APN. This resource should be defaulted to 2 retries. Should a device integrator wish to manipulate this the integrator should be able to do so.]]></Description>

```

Comment:
Since there is a default value, should this be optional resource? Same question applies to other resources with default values.